test/commands/generate/action_test.rb

```diff
@@ -42,6 +42,20 @@
       end
     end
 
+    it "handles nested routes correctly" do
+      with_temp_dir do |original_wd|
+        setup_container_app
+        command = Lotus::Commands::Generate::Action.new({}, 'web', 'admin/books#index')
+        capture_io { command.start }
+
+        assert_file_exists 'apps/web/templates/admin/books/index.html.erb'
+
+        expected_route = "get '/admin/books', to: 'admin/books#index'"
+        actual = File.read('apps/web/config/routes.rb')
+        actual.must_include(expected_route)
```

Comment:
You can use the `assert_file_includes` here too, like this:
```ruby
assert_file_includes("apps/web/config/routes.rb", "get '/admin/books', to: 'admin/books#index'")
```
This custom assertion can compare multiple strings or regex